src/benchmarklib/benchmark_runner.cpp

```diff
@@ -534,6 +534,7 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
     ("compression", "Specify vector compression as a string. Options: " + compression_strings_option, cxxopts::value<std::string>()->default_value(""))  // NOLINT(whitespace/line_length)
     ("chunk_indexes", "Create chunk indexes (separate index per chunk; columns defined by benchmark)", cxxopts::value<bool>()->default_value("false"))  // NOLINT(whitespace/line_length)
     ("table_indexes", "Create table indexes (index per table column; columns defined by benchmark)", cxxopts::value<bool>()->default_value("false"))  // NOLINT(whitespace/line_length)
+    ("index_everything", "Create indexes on every column of every table of the benchmark", cxxopts::value<bool>()->default_value("false"))  // NOLINT(whitespace/line_length)
```

Comment:
Will this be removed since we have defined indexes for each benchmark?

Reply:
Yes.